### PR TITLE
[ADPRODUCTS-370] Documentation updates with CS data revamp

### PR DESCRIPTION
**Pull request by skahmann3**
Creating Sponsored Tiles dataset documentation in DTMO. Also adding the new datatables to the docs.

### CODE REVIEW

**Comment by skahmann3** [comment]
@rebecca-burwei - created documentation for Tiles, noting the new datasetes. Please review when you get a chance! Though I'll hold off on merging until we know where the new tables are landing in Looker

File: src/datasets/other/sponsored tiles/sponsored_tiles.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Sponsored Tiles
+
+## Introduction
+
+Firefox Sponsored Tiles is an advertising-based feature available on the new tab of both desktop and mobile Firefox browsers. This feature populates the new tab's Top Sites section with up to two Sponsored Tiles. These Sponsored Tiles display a company logo and link to the company website. Mozilla's Merino serves the advertisements and may refresh the advertisers every 15 minutes. Each click on a Sponsored Tile generates revenue for Mozilla.
```
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:5]:
```suggestion
Firefox Sponsored Tiles is an advertising-based feature available on the new tab of both desktop and mobile Firefox browsers. This feature populates the new tab's Top Sites section with up to two Sponsored Tiles. These Sponsored Tiles display a company logo and link to the company website. Mozilla's Contile serves the advertisements and may refresh the advertisers every 15 minutes. Each click on a Sponsored Tile generates revenue for Mozilla.
```

File: src/datasets/other/sponsored tiles/sponsored_tiles.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Access to Sponsored Tiles data by advertiser is restricted to members of the contextual services working group. These restrictions are designed to protect user privacy, preventing excessive acess to data which links a given client to their interactions with different advertisers. For more information on requesting access, see [comment here](https://mana.mozilla.org/wiki/display/DATA/Data+Access+Policies).
+
+## Big Query Tables and Looker Explores
+
+### All-Mozilla access
+
+| Access-restriction(s)  | Big Query Table                                     | Looker Explore                         | Description                                 |
+| ---------------------------- | --------------------------------------------- | -------------------------------------- | ------------------------------------------- |
+| All Mozillians               | `telemetry.sponsored_tiles_clients_daily`     | TBD                                    | Workhorse dataset for Sponsored Tiles, includes desktop and mobile data. All new Sponsored Tiles metrics are added to this table. |
```
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:19]:
```suggestion
| All Mozillians               | `telemetry.sponsored_tiles_clients_daily`     | TBD                                    | Workhorse dataset for Sponsored Tiles, includes desktop and mobile data. All new Sponsored Tiles metrics are added to this table. Does not include advertiser data. |
```

File: src/datasets/other/sponsored tiles/sponsored_tiles.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| All Mozillians               | `telemetry.newtab_interactions`               | Firefox Desktop > New Tab Interactions | In-development dataset for basic analyses. Available metrics are limited to **desktop** clicks and impressions.                             |
+| All Mozillians               | `telemetry.newtab`                            | Firefox Desktop > Newtab               | Expanded newtab **desktop** dataset. Requires unnesting events. |
+| Contextual Services          | `contextual_services.event_aggregates`        | Contextual Services > Event Aggregates | Workhorse dataset for Sponsored Tiles and Suggest analyses by advertiser. |
+| Contextual Services          | `contextual_services_derived.adm_forecasting` | Contextual Services > AdM Forecasting  | Dataset with required components for Sponsored Tiles and Suggest revenue forecasts. |
+| Revenue | `contextual_services.event_aggregates` x  `revenue.revenue_data`   | Revenue > AdM Revenue with Telemetry  | Revenue information combined with usage metrics. This dataset is useful for CPC analyses. |
```
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:24]: I think this dataset limited to people in both Revenue and Contextual Services.

**Review by rebecca-burwei — COMMENTED** [review]
Some minor changes, but overall looks good.

**Review by rebecca-burwei — APPROVED** [review]

**Review by badboy — CHANGES_REQUESTED** [review]
Please use an underscore `_` instead of spaces in directory and file names.
Additionally this file is not referenced anywhere, and thus won't be accessible in the book.
You can add it to the table of contents in the `SUMMARY.md` file.

I think addressing these 2 things should then make the tests pass as well.

**Comment by skahmann3** [comment]
Thanks for catching those edits, @badboy. Will you re-review? But please don't merge changes yet, we're waiting for another BQ table to land before finalizing these updates

**Review by badboy — APPROVED** [review]
No blocker from my side then. Merge when you are ready.

**Review by rebecca-burwei — APPROVED** [review]
Looks good!

**Pull request merged**